process/sync/shardblock.go

```diff
@@ -877,3 +881,7 @@ func (boot *ShardBootstrap) getBlockBodyRequestingIfMissing(headerHandler data.H
 
 	return blockBody, nil
 }
+
+func (boot *ShardBootstrap) getCurrHeaderHash() []byte {
```

Comment:
no need for this wrapper. baseBootstrap can have access to blkc.

Reply:
removed